profiling/chrome-tracing/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+find_package(MPI REQUIRED)
+
+add_library(kp_chrome_tracing SHARED kp_chrome_tracing.cpp)
+
+if(NOT DEFINED USE_MPI OR USE_MPI)
```

Comment:
```suggestion
if(KOKKOSTOOLS_USE_MPI AND TARGET MPI::MPI_CXX)
```

Reply:
Thanks, applied this suggestion in deb807bd501adc8626736d9f2e9ff700c10c6ad8.
See MPI support discussion above.